docs/api-testing/vault.md

```diff
@@ -58,73 +58,50 @@ Then in your tests, you'd reference the variable by the `Key` using the followin
 
 Below is an example of how to create an Authentication Snippet.
-1. First, create a new test with a request component that requires basic authentication. For examples, check the [Sauce Labs REST API endpoints](/dev/api/) for ideas.<br/><img src={useBaseUrl('img/api-fortress/2021/04/exampleSnippetRequest.png')} alt="Example Snippet Request"/>
-1. Select to the **+ Add Request Headers** section below the request component.<br/><img src={useBaseUrl('img/api-fortress/2021/04/addRequestHeader.png')} alt="Add Request Header"/>
-1. Select **Basic Authentication** from the list.<br/><img src={useBaseUrl('img/api-fortress/2021/04/basicAuth.png')} alt="Basic Auth Component"/>
-1. Enter the details for `username` and `password`, then select **Save**.<br/><img src={useBaseUrl('img/api-fortress/2021/04/basicAuthDetails.png')} alt="Basic Auth Details Component"/>
-1. Once the **Authorization Header** appears, highlight it in the UI, then select the **Export snippet from selection** icon in the toolbar.<br/><img src={useBaseUrl('img/api-fortress/2021/04/authSnippet.png')} alt="Auth Snippet screenshot"/>
+1. First, create a new test with a request component that requires basic authentication. For examples, check the [Sauce Labs REST API endpoints](/dev/api/) for ideas.<br/><img src={useBaseUrl('img/api-testing/vault_exampleSnippetRequest.png')} alt="Example Snippet Request"/>
+1. Select **Add Child Component** below the request component.<br/><img src={useBaseUrl('img/api-testing/vault_addRequestHeader.png')} alt="Add Request Header"/>
+1. Select **Basic Authentication** from the list.<br/><img src={useBaseUrl('img/api-testing/vault_basicAuth.png')} alt="Basic Auth Component"/>
+1. Enter the details for `username` and `password`, then select **Confirm**.<br/><img src={useBaseUrl('img/api-testing/vault_basicAuthDetails.png')} alt="Basic Auth Details" width="600"/>
+1. Once the **Authorization Header** appears, click **Save Changes**.<br/>
+1. Hold down the Ctrl key, highlight the code in the UI and **Save Snippet**.<br/><img src={useBaseUrl('img/api-testing/vault_savesnippet.png')} alt="Save snippet"/>
+1. Give the snippet a name and **Save Snippet**.<br/><img src={useBaseUrl('img/api-testing/vault_snippetname.png')} alt="Snippet name" width="600"/>
 
 Consider a scenario where this login will be required for all the endpoints we have to test. It makes sense for this call to be stored in the **Vault**.
 
-Now you can choose to insert or invoke this snippet in future tests that require a Basic Authentication header.
+Now you can choose to **Call Snippet** or **Paste Snippet** in future tests that require the Login.
+<img src={useBaseUrl('img/api-testing/call_paste_snippet.png')} alt="Snippet options"/>
```

Comment:
img/api-testing/call_paste_snippet.pngcall_paste_snippet.png , the image is missing.

Reply:
@aliciakrebs added!